_core/smart/tutorial-counter.md

```diff
@@ -179,5 +182,5 @@ With the completion of this tutorial, you ...
 
 {:.no_toc}
 
-* <a href="principals.html">Guide: Understanding principals</a>
+* <a href="tutorial-deploy-call.html">Next tutorial: Deploying and interacting with smart contracts</a>
```

Comment:
this tutorial doesn't exist anymore with the changes

Reply:
Zapped